### ISSUE TITLE
How to update xySeries, if values come from another thread

### ISSUE THREAD

**Issue by permiakovsa** [HanSolo/charts#48]
Hello, i'd like to check the dynamic data performance, so i tried to add values from the other class. Values were edit, but not shown, i suppose i need to call something like - updataDataset. Is it possible?

**Comment by HanSolo** [comment]
Of course that‘s possible but please provide a code snippet that shows the problem so that I can reproduce it.

**Comment by permiakovsa** [comment]
I take LineChartTest and add presenters here, i tried several times. The values are added, but it is possible to show them, only after manual screen resize.

Here is presenters, first in Kotlin. It uses specific dispatcher to be ok, but its not even with runLater
```
class Presenter(xySeries1: XYSeries<XYItem>) {

    init {
        var x = 12.0
        CoroutineScope(Dispatchers.JavaFX).launch {
            while(true){
                println("Hello")
                delay(1000)
                Platform.runLater {
                    xySeries1.items.add(XYChartItem(++x, 160.0," "))
                }
            }
        }
    }
}
```

Then I check the java version of the same presenter with thread

```
public class Presenter2 {

    public Presenter2 (XYSeries xySeries){

        new Thread(new Runnable() {
            @Override
            public void run() {
                int x = 0;
                while (true){
                    try {
                        Thread.sleep(1000);
                    } catch (InterruptedException e) {
                        e.printStackTrace();
                    }
                    int finalX = x;
                    Platform.runLater(() -> xySeries.getItems().add(new XYChartItem(finalX, 160, "")));
                    x++;

                }
            }
        }).start();
    }
}

```
but it also didn't work

and i tried with the third presenter, that uses method of application
```
public class Presenter3 {

    public Presenter3(App application) {
        new Thread(new Runnable() {
            @Override
            public void run() {
                int x = 0;
                while (true) {
                    try {
                        Thread.sleep(1000);
                    } catch (InterruptedException e) {
                        e.printStackTrace();
                    }
                    Platform.runLater(() -> application.addValue());
                }
            }
        }).start();
    }
}
```
new application method:
```
   public void addValue() {
        Platform.runLater(() -> {
            xySeries1.getItems().add(new XYChartItem(x, 160, ""));
        });
        x++;
    }
```

**Comment by HanSolo** [comment]
Ok, now I see. Fixed it in the JDK16 branch with commit 39d8877842e8006d895948b4637c5c3ecb33e745. 
If I find time I will also backport it to 11.

**Comment by permiakovsa** [comment]
Thank you for the work!

**Comment by HanSolo** [comment]
Just for your info, I've released 11.8 which is more or less a backport of the jdk16 branch, meaning to say it contains a lot of changes and additional charts.
